malaria-app-android/src/main/res/layout/rapid_fire.xml

```diff
@@ -41,8 +41,7 @@
                     android:layout_weight="1"
                     android:gravity="center"
                     android:text="Score :"
-                    android:textColor="@color/golden_brown"
-                    android:textSize="@dimen/text_size_20sp"
+                    android:textColor="@color/textColorPrimary"                    android:textSize="@dimen/text_size_20sp"
```

Comment:
textsize can be on new line

Reply:
It happened while removing conflicts, I've corrected it locally, will include in next PR :+1:

Reply:
Okay.